pelias-config/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
             - ${DATA_DIR}:/data
             - ./blacklist/:/data/blacklist
     transit:
-        image: pelias/transit:master
+        image: milesgrantibigroup/ibi-transit:v5
```

Comment:
Does this tie the image tied to your account instead of the organization?

Reply:
We'd need to sign up as an organization. Have added an agenda item to discuss this week.